py-scripts/lf_create_vap_cv.py

```diff
@@ -383,8 +383,10 @@ def main():
                         help="Chamberview scenario name (by default: Automation")
     parser.add_argument("-vr", "--vap_radio", default="wiphy0",
                         help="vap radio name (by default: wiphy0")
-    parser.add_argument("-vf", "--vap_freq", default="2437",
```

Comment:
Is there a particular reason you removed the default value? I'm not against it, as you protect script execution with an error message, but customers who use this script with the default value presently will now encounter the error after this change (unsure if any but highlighting just in case).